src/App.scss

```diff
@@ -1,3 +1,3 @@
 iframe {
   display: none;
-}
+}
```

Comment:
Missing empty line at the end of the file

Reply:
> Missing empty line at the end of the file

Why do you upvote this comment @HolyRaveOS ? GitHub even provides this warning:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/63243136/6b3ff3c4-202a-4f07-a0df-a12018dca639)